mozilla-central/prefs_general.diff

```diff
@@ -5,6 +5,7 @@ diff --git a/testing/profiles/prefs_general.js b/testing/profiles/prefs_general.
 
 +// Make sure that activity stream doesn't hit the network.
 +user_pref("extensions.@activity-streams.telemetry", false);
++user_pref("extensions.@activity-streams.activateExperiments", false);
```

Comment:
Ah the dangers of manually editing patches.. `patch: **** malformed patch at line 12:  // Increase the APZ content response timeout in tests to 1 minute.`

You need to indicate there's 1 additional line added on line 4: `+268,7` to `+268,8`